board/aarch64/README.md

```diff
@@ -4,11 +4,39 @@ aarch64
 Board Specific Documentation
 ----------------------------
 
+- [Banana Pi R3](#banana-pi-r3)
 - [Marvell CN9130-CRB](cn9130-crb/)
 - [Microchip SparX-5i PCB135 (eMMC)](sparx5-pcb135/)
 - [NanoPi R2S](r2s/)
 - [Raspberry Pi 4 b](#raspberry-pi-4-b)
 
+# Banana Pi R3
+
+## Support level
+Full support for all Infix enabled features including switched ethernet ports, WiFi,
+and SFP interfaces. The board includes comprehensive hardware support for
+MediaTek MT7986 SoC features.
+
+### Hardware features
+The Banana Pi R3 is a high-performance networking board featuring:
+- MediaTek MT7986 ARM Cortex-A53 quad-core processor
+- 4x Gigabit LAN ports (lan1-lan4)
+- 1x Gigabit WAN port
+- 2x SFP ports (sfp1, sfp2) for fiber connectivity
+- Dual WiFi interfaces (wifi0 for 2.4GHz, wifi1 for 5GHz)
+- USB support
+- SD card boot support
+
+### Network configuration
+The board comes preconfigured with:
+- 4 switched LAN ports for internal networking
+- Dedicated WAN port with DHCP client enabled
+- SFP ports for high-speed fiber connections
+- Dual WiFi interfaces for wireless connectivity
+
+### Pre-built images
+SD card image: [infix-bpi-r3-sdcard.img](https://github.com/kernelkit/infix/releases/download/latest-boot/infix-bpi-r3-sdcard.img)
+
```

Comment:
as per previous discussions, per-board specifics should be be documented in $arch/$bord/README.md

Reply:
Please remember to reference the per-boar README's here though <3

Reply:
Fixed